app/modules/pagination/actions/pagination-fetch-all.js

```diff
@@ -0,0 +1,25 @@
+import {fetchPage} from "./pagination-fetch-page"
+import {paginationSetFetching} from "./pagination-set-fetching"
+import {paginationSetAssignmentURL} from "./pagination-set-assignment-url"
+import {nextPage} from "../selectors"
+
+// PUBLIC: Async thunk action for fetching all pages of repos
```

Comment:
:art: maybe this is just a coding style thing that we didn't cover in the review, but I'd recommend using [JSDoc](http://usejsdoc.org/) syntax on things that are the public API surface of the app, for a couple of reasons:

 - structured but not pedantic (note that I didn't both documenting the parameter below)
 - a healthy ecosystem of tooling knows how to parse this and help guide consumers of the API
 - encourages you to document things in a structured way

```js
/**
 * Fetch all pages of repositories associated with an assignment
 *
 * @return An asynchronous thunk action 
 */
export const fetchAllPages = (assignmentURL) => {

}
```

Reply:
Added JSDoc comments to all thunk actions, I presume we don't need them on just regular store actions?